README.md

```diff
@@ -1,47 +1,53 @@
 # Rekor Command Line Interface
 
-Early Development / Experimental use only.
-
 Rekór - Greek for “Record”
 
 Rekor's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain.  Rekor will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's life-cycle, based on signed metadata stored within a tamper proof binary (merkle) tree.
 
-The Rekor CLI requires a running instance of the [rekor-server](https://github.com/projectrekor/rekor-server).
+Rekor requires a running instance of the [rekor-server](https://github.com/projectrekor/rekor-server).
+
+## Sign your release
+
+Before using rekor, you are required to sign your release. For now we use GPG
+(we plant to extend to other signing formats in the foreseeable future)
```

Comment:
I was thinking about this a bit after the API writeup... it might be best to break out the various signature/key options into separate interfaces and reuse this code between server & client.

Reply:
Yes, we should make this a plug-able thing, where someone can use a signing system of their preference (if this is what you mean). We should also look at a similar framework for allowing ingestion of different manifest types (in-toto, sbom etc).